docs/stale-subvolume.md

```diff
@@ -0,0 +1,38 @@
+# subvolume cleanup
```

Comment:
Let's create a `/design` folder for design docs like this. Let's reserve the `/docs` folder for end-user documentation.

Reply:
Alternatively, perhaps no design doc is needed. This doc already looks like user documentation. Once feedback is stabilized, you could just go ahead and implement the command in this same PR.

Reply:
Yes, my though was same only, this doc is use doc(not design doc) and we'll implement  the command in this pr only.

Reply:
in that case, will go ahead and send a PR to implement it, we can later on merge it as a user document. I don't see any design doc for previous command too.

Reply:
no need for separate pr, you add a separate commit as user doc and implementation should be single pr.